### ISSUE TITLE
Crashes on android release

### ISSUE THREAD

**Issue by cyberpwnn** [builttoroam/device_calendar#273]
```
(io.flutter.embedding.engine.FlutterEngine@68e75b3).
[   +2 ms] E/AndroidRuntime(22998): FATAL EXCEPTION: main
[        ] E/AndroidRuntime(22998): Process: care.myguide.app, PID: 22998
[   +2 ms] E/AndroidRuntime(22998): java.lang.RuntimeException: Unable to start activity ComponentInfo{care.myguide.app/care.myguide.app.MainActivity}: java.lang.IllegalStateException: registrar.context() must not be null
[        ] E/AndroidRuntime(22998):     at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3385)
[        ] E/AndroidRuntime(22998):     at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3524)
[   +4 ms] E/AndroidRuntime(22998):     at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:83)
[   +1 ms] E/AndroidRuntime(22998):     at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:135)
[   +1 ms] E/AndroidRuntime(22998):     at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:95)
[        ] E/AndroidRuntime(22998):     at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2131)
[   +1 ms] E/AndroidRuntime(22998):     at android.os.Handler.dispatchMessage(Handler.java:107)
[   +1 ms] E/AndroidRuntime(22998):     at android.os.Looper.loop(Looper.java:214)
[   +2 ms] E/AndroidRuntime(22998):     at android.app.ActivityThread.main(ActivityThread.java:7707)
[   +1 ms] E/AndroidRuntime(22998):     at java.lang.reflect.Method.invoke(Native Method)
[   +1 ms] E/AndroidRuntime(22998):     at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:516)
[        ] E/AndroidRuntime(22998):     at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:950)
[   +1 ms] E/AndroidRuntime(22998): Caused by: java.lang.IllegalStateException: registrar.context() must not be null
[   +1 ms] E/AndroidRuntime(22998):     at com.builttoroam.devicecalendar.DeviceCalendarPlugin$Companion.registerWith(DeviceCalendarPlugin.kt:74)
[   +1 ms] E/AndroidRuntime(22998):     at com.builttoroam.devicecalendar.DeviceCalendarPlugin.registerWith(Unknown Source:2)
[   +3 ms] E/AndroidRuntime(22998):     at io.flutter.plugins.GeneratedPluginRegistrant.registerWith(GeneratedPluginRegistrant.java:20)
[        ] E/AndroidRuntime(22998):     at care.myguide.app.MainActivity.configureFlutterEngine(MainActivity.java:30)
[   +2 ms] E/AndroidRuntime(22998):     at io.flutter.embedding.android.FlutterActivityAndFragmentDelegate.onAttach(FlutterActivityAndFragmentDelegate.java:179)
[   +1 ms] E/AndroidRuntime(22998):     at io.flutter.embedding.android.FlutterActivity.onCreate(FlutterActivity.java:410)
[   +1 ms] E/AndroidRuntime(22998):     at android.app.Activity.performCreate(Activity.java:7825)
[        ] E/AndroidRuntime(22998):     at android.app.Activity.performCreate(Activity.java:7814)
[   +1 ms] E/AndroidRuntime(22998):     at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1325)
[   +3 ms] E/AndroidRuntime(22998):     at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3360)
[   +1 ms] E/AndroidRuntime(22998):     ... 11 more

```

**Comment by juampiq6** [comment]
having the same problem here, hope it gets fixed.
on debug mode its working perfectly

**Comment by juampiq6** [comment]
This issue was solved  on #99.
RIght here: https://github.com/builttoroam/device_calendar/issues/99#issuecomment-612449677_

**Comment by andzejsw** [comment]
Proguard related issue. Readme improved here: https://github.com/builttoroam/device_calendar/pull/287/commits/43868ab89630db7e4640f61067d0bf8dba18f786
